frontend/views/pages/Contributions.vue

```diff
@@ -2,167 +2,165 @@
     :isCard='true'
+    data-test='addIncomeDetailsFirstCard'
     :title='L("Add your income details")'
     :svg='SvgContributions'
+    v-if='!memberGroupProfile.incomeDetailsType'
   )
     i18n(tag='p') This will allow you to start receiving or giving mincome.
-    i18n(tag='button' @click='openModal("IncomeDetails")') Add income details
+    i18n(
+      tag='button'
+      data-test='openIncomeDetailModal'
+      @click='openModal("IncomeDetails")'
+    ) Add income details
 
-  section.card
-    i18n(tag='h2' class='card-header') Receiving
+  template(v-else)
+    .c-contribution-header
+      .has-text-1
+        i18n(
+          v-if='needsIncome'
+          tag='p'
+          data-test='headerNeed'
+          :args='{ amount: `<span class="has-text-bold has-text-0">${upTo}</span>` }'
+        ) You need {amount}
+        i18n(
+          v-else
+          tag='p'
+          data-test='headerPledge'
+          :args='{ upTo: `<span class="has-text-bold has-text-0">${upTo}</span>` }'
+        ) You are pledging up to {upTo}
 
-    ul.c-ul
-      contribution(
-        v-for='(contribution, index) in fakeStore.receiving.nonMonetary'
-        :key='`contribution-${index}`'
-      )
-        //- TODO: verify this is safe (no XSS)
-        span(v-html='textReceivingNonMonetary(contribution)')
-        text-who(:who='contribution.who')
-
-      contribution(
-        v-if='doesReceiveMonetary'
-        variant='editable'
-        ismonetary=''
-        @interaction='handleFormTriggerClick'
-      )
-        //- TODO: verify this is safe (no XSS)
-        span(v-html='textReceivingMonetary(fakeStore.receiving.monetary)')
-        text-who(:who='fakeStore.groupMembersPledging')
-        i18n each month
-
-  section.card
-    i18n(tag='h2' class='card-header') Giving
-    ul
-      contribution.has-text-weight-bold(
-        v-for='(contribution, index) in fakeStore.giving.nonMonetary'
-        :key='`contribution-${index}`'
-        variant='editable'
-        @new-value='(value) => handleEditNonMonetary(value, index)'
-      ) {{contribution}}
-
-      contribution(
-        variant='unfilled'
-        @new-value='submitAddNonMonetary'
-      )
-        i.icon-heart(aria-hidden='true')
-        i18n Add a non-monetary method
+        i18n(
+          tag='p'
+          :args='{ paymentMethod: `<span class="has-text-bold has-text-0">${paymentMethod}</span>` }'
+        ) Payment method {paymentMethod}
 
-    contribution(
-      v-if='doesGiveMonetary'
-      variant='editable'
-      ismonetary=''
-      @interaction='handleFormTriggerClick'
-    )
-      //- TODO: use displayWithCurrency!
-      i18n(
-        :args='{amount:`${fakeStore.currency}${fakeStore.giving.monetary}`}'
-      ) Pledging up to {amount}
-      i18n to other's mincome
-      i18n(
-        v-if='fakeStore.giving.monetary == 0' :args='{amount: "[$170]"}'
-        tag='p'
-      ) (The group's average pledge is {amount})
+      div
+        i18n(
+          tag='button'
```

Comment:
Is this outside `div` necessary or is there a way to add a class or something to the i18n tag so that it's not necessary?